docker/postgres/01-init-user.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -e
+
+psql -v ON_ERROR_STOP=1 --username postgres --dbname postgres <<-EOSQL
```

Comment:
Rename PostgreSQL Database to geostore because it's the database name used in the official documentation
```suggestion
psql -v ON_ERROR_STOP=1 --username postgres --dbname geostore <<-EOSQL
```